main/templates/post/post_update.html

```diff
@@ -22,6 +22,9 @@
       </div>
       <div class="col-sm-6">
         {{forms.select_field(form.language_key)}}
+        <span>Missing a language here? <a class="api-url"
+                                          href="https://github.com/welovecoding/vote4code/issues/new"
+                                          target="_blank">Submit it.</a></span>
```

Comment:
```html
<p class="help-block">
  Missing a language here? 
  <a href="https://github.com/welovecoding/vote4code/issues/new" target="_blank">
    Submit it.
  </a>
</p>
```

No need for class if we are not _coloring_ it :)

(Not sure how it will work with `help-text` but it should be ok.. Bootstrap's stuff)